README.md

```diff
@@ -43,9 +51,6 @@ Bits requires the following peer dependencies to be installed in the host applic
 
 ```
 classnames: ^2.2.5
-react: ^0.14.9 || ^15.0.0 || ^16.0.0
```

Comment:
Why have these vanished? Or perhaps why were they there in the first place?

Reply:
They used to be peer dependencies, in that the application that consumes Bits would be responsible for actually including them. Now we're using Storybook, running Bits in isolation absolutely requires them, so they're no longer peer deps. We don't need to advise their additional installation in the README because they'll be installed when `npm i` is executed.

Reply:
They are now `dependencies` rather than `peerDependencies` because they're now needed in this app for storybook to run.

They were there originally because they weren't need in this app, but would be needed in any app that consumed this.

Reply:
Great - thank you!